libnavui-dropin/src/main/res/layout/drop_in_navigation_view.xml

```diff
@@ -48,34 +59,37 @@
                 android:layout_margin="8dp"
                 app:layout_constraintTop_toBottomOf="@id/guidanceLayout"
                 app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toEndOf="parent"/>
-        </androidx.constraintlayout.widget.ConstraintLayout>
+                app:layout_constraintEnd_toEndOf="parent"
+                tools:layout_height="200dp"
+                tools:background="#eee" />
 
-        <View
```

Comment:
I removed this because view drop shadow should be done either by implementing [Outline](https://developer.android.com/reference/android/graphics/Outline) or by setting correct [elevation](https://material.io/design/environment/elevation.html#default-elevations). I've used 8dp elevation as suggested in Material Design guide.